corehq/apps/case_importer/templates/case_importer/import_cases.html

```diff
@@ -2,6 +2,53 @@
 {% load report_tags %}
 {% load i18n %}
 
+{% block js-inline %}
+<script id="caseUploadsResource" type="text/plain">{% url 'case_importer_uploads' domain %}</script>
```

Comment:
This is interesting, don't think I've ever seen this pattern before. Why is it better than just assigning the URL to a var directly?